README.md

```diff
@@ -467,8 +478,14 @@ dependency: # <9>
 6. The `paths` are the path list that will be checked (and fixed) by license-eye, default is `['**']`. Formats like `**/*`.md and `**/bin/**` are supported.
 7. The `paths-ignore` are the path list that will be ignored by license-eye. By default, `.git` and the content in `.gitignore` will be inflated into the `paths-ignore` list.
 8. On what condition License-Eye will comment the check results on the pull request, `on-failure`, `always` or `never`. Options other than `never` require the environment variable `GITHUB_TOKEN` to be set.
-9. `dependency` section is configurations for resolving dependencies' licenses.
-10. `files` are the files that declare the dependencies of a project, typically, `go.mod` in Go project, `pom.xml` in maven project, and `package.json` in NodeJS project. If it's a relative path, it's relative to the `.licenserc.yaml`.
+9. The `license-location-threshold` specifies the index threshold where the license header can be located.
+10. The `language` is an optional configuration. You can set the language license header comment style, if it doesn't exist, it will use the default configuration at the `languages.yaml`.
```

Comment:
```suggestion
10. The `language` is an optional configuration. You can set the language license header comment style. If it doesn't exist, it will use the default configuration at the `languages.yaml`. An example is to use block comment style for Go codes. (Link to the example configuration)
```

Reply:
> 

Hi @kezhenxu94 , Thanks for your reply, I will fix it.

Reply:
Hi @kezhenxu94 , I fix the content, please review it again, Thanks.